### ISSUE TITLE
404 login/does not work with /v2?

### ISSUE THREAD

**Issue by flaccid** [aptible/docker-registry-proxy#2]
```
$ docker login registry.foo.bar
Username: flaccid
Password:
Email:
Error response from daemon: Login: 404 page not found
 (Code: 404; Headers: map[Server:[nginx/1.6.3] Date:[Thu, 09 Jul 2015 12:59:34 GMT] Content-Type:[text/plain; charset=utf-8] Content-Length:[19] Connection:[keep-alive] Docker-Distribution-Api-Version:[registry/2.0]])
```

I'm running the standard registry:2 from docker.

**Comment by flaccid** [comment]
Fixed in https://github.com/aptible/docker-registry-proxy/pull/3.

**Comment by aaw** [comment]
Fixed in #5, which includes changes from #3. Thanks again, @flaccid!
